CHANGELOG-developer.next.asciidoc

```diff
@@ -68,6 +68,8 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 - Split split httpmon out of x-pack/filebeat/input/internal/httplog. {pull}36385[36385]
 - Beats publishing pipeline does not propagate the close signal to its clients any more. It's responsibility of the user to close the pipeline client. {issue}38197[38197] {pull}38556[38556]
 - Debug log entries from the acker (`stateful ack ...` or `stateless ack ...`) removed. {pull}39672[39672]
+- Journald input now calls `journalctl` instead of using `github.com/coreos/go-systemd/v22@v22.5.0/sdjournal`, the CGO dependency has been removed from Filebeat {pull}40061[40061]
+- Filebeat now requires `fork` and `exec` syscalls to call `journalctl` {pull}40061[40061]
```

Comment:
You should specifically say that you added `fork` and `exec` to the default seccomp policy.

It may be nice to isolate adding those syscalls to the list to only the cases where journalctl is actually used.